src/libs/portfolio/interfaces.ts

```diff
@@ -17,6 +17,10 @@ export type TokenResult = Omit<CustomToken, 'standard'> & {
   amount: bigint
   simulationAmount?: bigint
   amountPostSimulation?: bigint
+  availableAmount?: bigint
+  cashback?: bigint
+  saved?: bigint
+  shouldPopsUpConfetti?: boolean
```

Comment:
Let's sync up these new flags with @PetromirDev and @gergana95 , who have a better context on the TokenResult interface.

Reply:
We can create a new GasTankToken type and not add them into the TokenResult or in object gasTankData which whould be only with onGasTank tokens? @PetromirDev what do you think?